app/src/main/java/com/example/drawingApp/CustomViews/ForceEditText.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.drawingApp.CustomViews
+
+import android.content.Context
+import android.text.TextWatcher
+import android.util.AttributeSet
+import android.widget.EditText
+import kotlin.math.max
+import kotlin.math.min
+
+class ForceEditText @JvmOverloads constructor(
```

Comment:
Also shouldn't be in this ticket...

Reply:
I think this was due to previous conflicts